lib/usd/ui/views/USDImportDialog.cpp

```diff
@@ -36,14 +37,29 @@ USDImportDialog::USDImportDialog(const std::string& filename, QWidget* parent)
 {
 	if (!fStage)
 		throw std::invalid_argument("Invalid filename passed to USD Import Dialog");
+	if (mayaQtUtil == nullptr)
+		throw std::invalid_argument("Invalid IMayaMQtUtil passed to USD Import Dialog");
 
 	fUI->setupUi(this);
 
+	// If we were given some import data we will only use it if it matches our
+	// input filename. In the case where the user opened dialog, clicked Apply and
+	// then reopens the dialog we want to reset the dialog to the previous state.
+	const ImportData* matchingImportData = nullptr;
```

Comment:
This happens when a user opens the import dialog, sets some variants, changes prim (by checking) to import and clicks "Apply" to close dialog. Then before importing they reopen dialog (clicking Hierarchy View). We will use the import data to set the dialog to it's previous state.